dictionary/Dictionary.js

```diff
@@ -45,6 +45,24 @@ export default class Dictionary {
 	 *         evaluated using the provided parameters.
 	 */
 	get(key, parameters = {}) {}
+
+	/**
+	 * Tests whether the specified localization phrase exists in dictionary
```

Comment:
The wording should be "in the dictionary". I should have noticed this sooner, sorry.
Also, please terminate the sentence using a period ("...exists in the dictionary.").